index.js

```diff
@@ -16,159 +16,221 @@ const isDeflate = require('is-deflate')
 const encodingNegotiator = require('encoding-negotiator')
 
 function compressPlugin (fastify, opts, next) {
-  fastify.decorateReply('compress', compress)
```

Comment:
Can you set this to `null` instead? In this way it will be added to the prototype

Reply:
Thanks, added it back in and set it to null so added on the prototype later on the onrequest hook